### PR TITLE
DOC-8994 Couchbase Server Processes on Windows

### PR DESCRIPTION
**Pull request by RayOffiah**
Added new table for Windows processes

### CODE REVIEW

File: modules/install/pages/server-processes.adoc [hunk 1]
```diff
@@ -142,3 +142,95 @@ Instance count is configured in UI.
 | Analytics
 | _/opt/couchbase/bin/_
 |===
+
+== Windows
+
+The following table lists the Couchbase processes that run on the Windows platforms.
+
+.Couchbase Processes on Linux
```
Comment (bkumaran) [inline modules/install/pages/server-processes.adoc:150]: I see a few processes missing. This is from our list. Would be worth while to check with latest CB version.

WIN_PROCESSES_SPAWNED = ["backup.exe", "cbas.exe", "cbft.exe", "cbq-engine.exe",
                         "erl.exe", "eventing-producer.exe", "goxdcr.exe",
                         "indexer.exe", "java.exe", "memcached.exe",
                         "projector.exe", "prometheus.exe", "saslauthd-port.exe",
                         "eventing-consumer.exe"]

Comment (RayOffiah) [reply modules/install/pages/server-processes.adoc:150]: Where does that list come from?

**Comment by RayOffiah** [comment]
@bkumaran <-- I'm not familiar with Windows. How did you get that list of WIN_PROCESS_SPAWNED?

**Comment by bkumaran** [comment]
We actually installed CB server on windows and got the list. Better to do it again and get the latest list.

**Comment by RayOffiah** [comment]
Probably much easier if you just give me the list.

**Comment by bkumaran** [comment]
Updated the bug with details.

File: modules/install/pages/server-processes.adoc [hunk 2]
```diff
@@ -142,3 +142,121 @@ Instance count is configured in UI.
...
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `projector.exe`
+| Extracts secondary key from documents
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `prometheus.exe`
+| Metrics engine used by Analytics
```
Comment (bkumaran) [inline modules/install/pages/server-processes.adoc]: AFAIK this isn't specific to analytics 

Metrics engine used by Couchbase to store all stats
This is owned by ns_serv I believe.

Comment (RayOffiah) [reply modules/install/pages/server-processes.adoc]: Fixed

**Review by bkumaran — APPROVED** [review]

**Pull request merged**